installation.md

```diff
@@ -8,9 +8,9 @@ To use clangd, you need:
 
 ## Installing clangd
 
-You'll want a **recent** version of clangd. The current release is 13.0.
+For best results, use a recent version of clangd. The current release is 15.0.
```

Comment:
I would suggest pointing to https://github.com/clangd/clangd/releases/latest or https://releases.llvm.org/ instead of explicitly mentioning version number.

Reply:
Good idea, done